devel/CoreOSMakefiles/Portfile

```diff
@@ -12,10 +13,14 @@ long_description        These are some makefiles which are required to build \
                         implies MAKEFILEPATH=${prefix}/share/Makefiles
 
 homepage                http://opensource.apple.com/source/${name}/
-master_sites            http://opensource.apple.com/tarballs/${name}/
+master_sites            https://github.com/apple-oss-distributions/${name}/archive
 
-checksums               rmd160  ab3816588c11c9c9ef6720717d323846d69ebefe \
-                        sha256  6b8a9521bb65a2868181696f12515eb4c2498fb05a8f28075e29cb1eda67bae6
+checksums               rmd160  aa7a5724907a0719ea6c1c39c96ad2a11adaec56 \
+                        sha256  b8bf4c3fa727d71615665322bb22813eda15962544df5238cc37aa606bda276c \
+                        size    17200
+
+dist_subdir             apple-oss-distributions
```

Comment:
Why would we want all these ports to share a dist_subdir?

Reply:
6d86c28 has instead:
```tcl
# simulate stealth update; remove with next version change
dist_subdir             ${name}/${version}_1
extract.rename          yes
```

Reply:
sames here.